### PR TITLE
Fixes #28 Menu component

### LINKED ISSUE #28: Menu component
Similar to this:

image

So that the dropdown also looks more like it:
image

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

**Review by thabti — APPROVED** [review]

File: src/Menu/MenuItem.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import Link from 'react-router/lib/Link';
+import styles from './style.scss';
+
+const MenuItem = ({ children, icon, href, onClick }) => {
+  const Type = href ? Link : 'a';
+
+  return (
+    <li className={styles.item}>
```
Comment (thabti) [inline src/Menu/MenuItem.js]: can we allow users to pass their own `classNames` and others? so minor extensions and adaptation can happen?

File: src/Menu/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import styles from './style.scss';
+
+export { default as MenuItem } from './MenuItem.js';
+
+const Menu = ({ children, bordered }) => (
+  <ul className={`${styles.list} ${bordered && styles.bordered}`}>
```
Comment (thabti) [inline src/Menu/index.js]: allow custom classNames, and other props?
```
 <ul className={`${styles.list} ${bordered && styles.bordered} ${className}`} {...props} >
```

thoughts on allowing this escape hatch?

File: src/Menu/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import styles from './style.scss';
+
+export { default as MenuItem } from './MenuItem.js';
```
Comment (thabti) [inline src/Menu/index.js:4]: love this

**Review by thabti — APPROVED** [review]
I left a few comments.

File: src/Menu/index.js (hunk 3 above)
Comment (mmahalwy) [reply src/Menu/index.js:4]: Got it from you ;)

**Pull request merged**